pycbc/coordinates.py

```diff
@@ -78,7 +78,8 @@ def cartesian_to_spherical_polar(x, y, z):
     theta : {numpy.array, float}
         The polar angle.
     """
-    return numpy.arccos(z / cartesian_to_spherical_rho(x, y, z))
+    rho = cartesian_to_spherical_rho(x, y, z)
+    return numpy.arccos(z / rho) if rho else 0.0
```

Comment:
@a-r-williamson This doesn't look like it would work with vector import, right? 

@cdcapano Can verify, but I think where possibly we want to avoid assuming scalar or vector input on these kind of generic functions.

Reply:
Yes I agree, I'll rethink this. Perhaps something like:
```
return numpy.arccos(np.divide(z, rho, out=np.zeros_like(a), where=rho!=0.0))
```